h2o-py/tests/testdir_javapredict/pyunit_pubdev_4628_pubdev_4950_deeplearning_mojo_pojo_large.py

```diff
@@ -33,7 +33,14 @@ def runComparisonTests(autoEncoder, probleyType):
     test = df[:NTESTROWS, :]
     x = list(set(df.names) - {"response"})
 
-    deeplearningModel = build_save_model(params, x, train) # build and save mojo model
+    if autoEncoder:
+        try:
+            deeplearningModel = build_save_model(params, x, train) # build and save mojo model
+        except Exception as err:
```

Comment:
Please try to detect that the model was unstable and there was no other cause why the test failed

Reply:
Done!